azurelinuxagent/ga/exthandlers.py

```diff
@@ -1436,3 +1466,6 @@ def is_update_with_install(self):
         if update_mode is None:
             return True
         return update_mode.lower() == "updatewithinstall"
+
+    def is_continue_on_update_failure(self):
+        return self.data['handlerManifest'].get('continueOnUpdateFailure', False)
```

Comment:
Added a new flag `continueOnUpdateFailure` to specify weather to continue of update failure or not